dashboard/app/controllers/maker_controller.rb

```diff
@@ -1,28 +1,6 @@
 class MakerController < ApplicationController
   # Maker Toolkit is currently used in standalone Create Devices with Apps unit.
-  # Retrieves the relevant Create Devices with Apps unit version based on self.maker_script.
-  def home
-    # Redirect to login if not signed in
-    authenticate_user!
-
-    maker_unit_for_user = MakerController.maker_script current_user
-    current_level = current_user.next_unpassed_progression_level(maker_unit_for_user)
-    @maker_unit = {
-      assignableName: data_t_suffix('script.name', maker_unit_for_user[:name], 'title'),
-      lessonName: current_level.lesson.localized_title,
-      linkToOverview: script_path(maker_unit_for_user),
-      linkToLesson: script_next_path(maker_unit_for_user, 'next')
-    }
-  end
-
-  # TODO: deprecate this page which is no longer active since the Maker app was deprecated.
-  # For now, since this update https://github.com/code-dot-org/code-dot-org/pull/64729,
-  # maker scripts no longer are accessible via "family_name" of "devices".
-  # Temporarily show 'devices-2024' script.
-  def self.maker_script(for_user)
-    Unit.find_by(name: 'devices-2024')
-  end
-
+  # setup is a Maker-related support page.
   def setup
   end
```

Comment:
For my own understanding: Does it make sense to have an empty method in the controller?

Reply:
Yes, this is necessary as the route is mapped to 'maker/setup' in `routes.rb` - previous discussion at https://github.com/code-dot-org/code-dot-org/pull/64773#discussion_r2010966323. Whenever we get a GET request to studio.code.org/maker/setup, the route in routes.rb invokes the MakerController action `setup`. Via`webpackEntryPoints`, `setup.js` is triggered which renders `SetupGuide`.

Here's another example of an empty controller method - https://github.com/code-dot-org/code-dot-org/blob/c4fc12a65bfa74f770e453f14f0fc3ad2730a6e8/dashboard/app/controllers/admin_reports_controller.rb#L11-L12
Thanks for reviewing!